zuul.d/openvswitch-openvswitch-jobs.yaml

```diff
@@ -52,45 +52,49 @@
       ansible_collections_repo: github.com/ansible-collections/openvswitch.openvswitch
 
 
-# ansible-ee
+# ansible-test
 - job:
-    name: ansible-ee-integration-ovs-latest
-    parent: ansible-ee-tests
+    name: ansible-test-integration-ovs-latest
+    parent: ansible-test-integration-base
     pre-run:
-      - playbooks/ansible-ee/integration/pre.yaml
-      - playbooks/ansible-ee/openvswitch/pre.yaml
-    run: playbooks/ansible-ee/integration/run.yaml
+      - playbooks/ansible-network-openvswitch-appliance/pre.yaml
+    run: playbooks/ansible-network-openvswitch-appliance/run.yaml
```

Comment:
This run.yaml is actually empty.
You should instead run `playbooks/ansible-test-base/run.yaml`